solutions/devsprint-pedro-menezes-1/DeliveryAppChallenge/Modules/Product/Scenes/RodrigoLemos/EmailUseCase.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  EmailUseCase.swift
+//  DeliveryAppChallenge
+//
+//  Created by Rodrigo Lemos on 27/07/22.
+//
+
+import Foundation
+
+struct EmailUseCase {
+
+    func textDoNotContainsEmailRequirements(emailText: String) -> Bool {
```

Comment:
nome ficou confuso
acredito que essa func e a func debaixo tem a mesma responsabilidade
que é verificar se o email é valido
no caso, a func poderia chamar tipo: `func isEmailValid(for text: String) -> Bool`